deku/.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+{
+    "files.autoSave": "afterDelay",
+    "editor.formatOnSave": true,
+    "editor.formatOnPaste": true,
+    "editor.tabSize": 2,
+    "git.autofetch": true,
+    "ocaml.sandbox": {
+        "kind": "esy",
+        "root": "${workspaceFolder:deku}"
+    },
+    "indentRainbow.ignoreErrorLanguages": [
+        "*"
+    ],
+    "files.insertFinalNewline": true,
+    "terminal.integrated.commandsToSkipShell": [
```

Comment:
isn't this should be specific to one's workflow?

Reply:
Inserting the final newline is part of marigold style, is that what you were talking about?

Reply:
```
"terminal.integrated.commandsToSkipShell": [
```